cypress/package-lock.json

```diff
@@ -22,8 +22,9 @@
     },
     "node_modules/@ampproject/remapping": {
       "version": "2.2.1",
-      "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.1.tgz",
+      "resolved": "https://artifactory.tools.saucelabs.net:443/artifactory/api/npm/all-npm/@ampproject/remapping/-/remapping-2.2.1.tgz",
```

Comment:
This must not be the internal registry.
Same applies to all that file.

Reply:
fixed